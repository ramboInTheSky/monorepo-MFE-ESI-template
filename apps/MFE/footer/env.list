REACT_APP_CDN_BASEURL=https://xcdn.amido.com/content
REACT_APP_API_BASEURL=http://infra-demo.deploy.systems.amido/api
