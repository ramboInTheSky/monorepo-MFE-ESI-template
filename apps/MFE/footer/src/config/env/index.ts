const INTERNAL = Object.freeze({
    PRODUCTION: process.env.NODE_ENV === "production",
    DEVELOPMENT: process.env.NODE_ENV === "development",
    DEV_URL_OVERRIDE: process.env.DEV_URL_OVERRIDE || "",
    NODE_ENV: process.env.NODE_ENV,
    REACT_APP_BLOB_STORAGE_PATH: "/static-content",
})

const ENV_VARS = Object.freeze({
    REACT_APP_BLOB_STORAGE_SSR_BASEURL: process.env.REACT_APP_BLOB_STORAGE_SSR_BASEURL || "http://localhost:3002",
    PORT: process.env.PORT || 3002,
    REACT_APP_CDN_BASEURL: process.env.REACT_APP_CDN_BASEURL,
    REACT_APP_API_BASEURL: process.env.REACT_APP_API_BASEURL,
    POD_NAME: process.env.POD_NAME,
    LOG_LEVEL: process.env.LOG_LEVEL,
    REACT_APP_APPINSIGHTS_KEY: process.env.REACT_APP_APPINSIGHTS_KEY || "404aa502-76dd-4b93-ae84-46c453774652",
    REACT_APP_ENTRYPOINT: process.env.REACT_APP_ENTRYPOINT || "/footer",
    ASSETS_PATH: process.env.ASSETS_PATH || "/footerstatic",
    USE_LOCAL_STATIC_CONTENT: process.env.USE_LOCAL_STATIC_CONTENT || "true",
})

const VARS = {...INTERNAL, ...ENV_VARS}

export default VARS
