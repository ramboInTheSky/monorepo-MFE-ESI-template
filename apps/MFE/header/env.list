REACT_APP_CDN_BASEURL=https://xcdn.amido.com/content
REACT_APP_API_BASEURL=http://abc123.com
