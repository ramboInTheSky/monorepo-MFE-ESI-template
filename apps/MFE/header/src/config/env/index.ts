const INTERNAL = Object.freeze({
    PRODUCTION: process.env.NODE_ENV === "production",
    DEVELOPMENT: process.env.NODE_ENV === "development",
    DEV_URL_OVERRIDE: process.env.DEV_URL_OVERRIDE || "",
    NODE_ENV: process.env.NODE_ENV,
    REACT_APP_BLOB_STORAGE_PATH: "/static-content",
    USEDEVESI: process.env.USEDEVESI,
})

const ENV_VARS = Object.freeze({
    REACT_APP_BLOB_STORAGE_SSR_BASEURL: process.env.REACT_APP_BLOB_STORAGE_SSR_BASEURL || "http://localhost:3004",
    PORT: process.env.PORT || 3004,
    REACT_APP_CDN_BASEURL: process.env.REACT_APP_CDN_BASEURL
        ? process.env.REACT_APP_CDN_BASEURL
        : "https://xcdn.amido.com/content",
    REACT_APP_API_BASEURL: process.env.REACT_APP_API_BASEURL
        ? process.env.REACT_APP_API_BASEURL
        : "http://sandbox.eun.deploy.systems.next/api/header",
    POD_NAME: process.env.POD_NAME,
    REACT_APP_SERVE_PATH_PREFIX: process.env.REACT_APP_SERVE_PATH_PREFIX || "",
    ASSETS_PATH: process.env.ASSETS_PATH || "/headerstatic",
    LOG_LEVEL: process.env.LOG_LEVEL,
    FNC_NAME: process.env.FNC_NAME,
    REACT_APP_APPINSIGHTS_KEY: process.env.REACT_APP_APPINSIGHTS_KEY,
    BLOOMREACH_BASE_URL: process.env.BLOOMREACH_BASE_URL || "https://brm-suggest-0.brsrvr.com/api/v1/suggest/",
    REACT_APP_MEGANAV_BASEURL:
        process.env.REACT_APP_MEGANAV_BASEURL ||
        (INTERNAL.DEVELOPMENT || INTERNAL.USEDEVESI ? "http://localhost:3005" : ""),
    REACT_APP_MEGANAV_ASSETS_PATH: process.env.REACT_APP_MEGANAV_ASSETS_PATH || "/meganavstatic",
    GEOLOCATION_BASEURL:
        process.env.REACT_APP_GEOLOCATION_BASEURL || "https://services-uat.test.ecomm.systems.next/geolocation",
    ENVIRONMENT: process.env.ENVIRONMENT || "dev",
    REACT_APP_USE_TIME_MACHINE_COOKIE: process.env.REACT_APP_USE_TIME_MACHINE_COOKIE || "false",
    USE_LOCAL_STATIC_CONTENT: process.env.USE_LOCAL_STATIC_CONTENT || "true",
})

const VARS = {...INTERNAL, ...ENV_VARS}

export default VARS
