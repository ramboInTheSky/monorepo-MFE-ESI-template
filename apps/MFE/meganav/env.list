REACT_APP_CDN_BASEURL=https://xcdn.amido.com/content
REACT_APP_API_BASEURL=http://infra-demo.deploy.systems.next/api
REACT_APP_PLATMOD_CDN_BASEURL=/static-content
REACT_APP_BLOB_STORAGE_SSR_BASEURL=https://ecmbrowsefdsxeuw.azurefd.net