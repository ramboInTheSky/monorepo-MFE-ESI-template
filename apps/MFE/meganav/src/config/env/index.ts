const INTERNAL = Object.freeze({
    PRODUCTION: process.env.NODE_ENV === "production",
    DEVELOPMENT: process.env.NODE_ENV === "development",
    REACT_APP_DEV_URL_OVERRIDE: process.env.REACT_APP_DEV_URL_OVERRIDE || "",
    NODE_ENV: process.env.NODE_ENV,
    REACT_APP_BLOB_STORAGE_PATH: "/static-content",
})

const ENV_VARS = Object.freeze({
    REACT_APP_BLOB_STORAGE_SSR_BASEURL: process.env.REACT_APP_BLOB_STORAGE_SSR_BASEURL || "http://localhost:3005",
    PORT: process.env.PORT || 3005,
    REACT_APP_CDN_BASEURL: process.env.REACT_APP_CDN_BASEURL
        ? process.env.REACT_APP_CDN_BASEURL
        : "https://xcdn.amido.com/content",
    REACT_APP_API_BASEURL: process.env.REACT_APP_API_BASEURL
        ? process.env.REACT_APP_API_BASEURL
        : "http://sandbox.eun.deploy.systems.next/api/meganav",
    POD_NAME: process.env.POD_NAME,
    LOG_LEVEL: process.env.LOG_LEVEL,
    REACT_APP_APPINSIGHTS_KEY: process.env.REACT_APP_APPINSIGHTS_KEY,
    REACT_APP_SERVE_PATH_PREFIX: process.env.REACT_APP_SERVE_PATH_PREFIX || "",
    REACT_APP_USE_TIME_MACHINE_COOKIE: process.env.REACT_APP_USE_TIME_MACHINE_COOKIE || "false",
    ASSETS_PATH: process.env.ASSETS_PATH || "/meganavstatic",
    USE_LOCAL_STATIC_CONTENT: process.env.USE_LOCAL_STATIC_CONTENT || "true",
})

const VARS = {...INTERNAL, ...ENV_VARS}

export default VARS
