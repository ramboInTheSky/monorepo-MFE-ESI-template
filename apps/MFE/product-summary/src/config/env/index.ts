const INTERNAL = Object.freeze({
    PRODUCTION: process.env.NODE_ENV === "production",
    DEVELOPMENT: process.env.NODE_ENV === "development",
    DEV_URL_OVERRIDE: process.env.DEV_URL_OVERRIDE || "",
    NODE_ENV: process.env.NODE_ENV,
    REACT_APP_BLOB_STORAGE_PATH: "/static-content",
    REACT_APP_ENABLE_CYPRESS_SETTINGS: process.env.REACT_APP_ENABLE_CYPRESS_SETTINGS || false,
})

const ENV_VARS = Object.freeze({
    REACT_APP_BLOB_STORAGE_SSR_BASEURL: process.env.REACT_APP_BLOB_STORAGE_SSR_BASEURL || "http://localhost:3001",
    PORT: process.env.PORT || 3001,
    REACT_APP_CDN_BASEURL: process.env.REACT_APP_CDN_BASEURL
        ? process.env.REACT_APP_CDN_BASEURL
        : "https://xcdn.amido.com/content",
    REACT_APP_API_BASEURL: process.env.REACT_APP_API_BASEURL
        ? process.env.REACT_APP_API_BASEURL
        : "http://sandbox.eun.deploy.systems.next/api/product-summary",
    POD_NAME: process.env.POD_NAME,
    REACT_APP_SERVE_PATH_PREFIX: process.env.REACT_APP_SERVE_PATH_PREFIX || "",
    ASSETS_PATH: process.env.ASSETS_PATH || "/productsummarystatic",
    LOG_LEVEL: process.env.LOG_LEVEL,
    FNC_NAME: process.env.FNC_NAME,
    REACT_APP_APPINSIGHTS_KEY: process.env.REACT_APP_APPINSIGHTS_KEY,
    REACT_APP_USE_TIME_MACHINE_COOKIE: process.env.REACT_APP_USE_TIME_MACHINE_COOKIE || "false",
    ENVIRONMENT_NAME: process.env.ENVIRONMENT_NAME,
    USE_LOCAL_STATIC_CONTENT: process.env.USE_LOCAL_STATIC_CONTENT || "true",
})

const VARS = {...INTERNAL, ...ENV_VARS}

export default VARS
