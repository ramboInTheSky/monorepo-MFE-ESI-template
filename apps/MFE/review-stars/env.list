REACT_APP_CDN_BASEURL=https://next-sandbox.azureedge.net
REACT_APP_API_BASEURL=http://sandbox.eun.deploy.systems.next/api/review-stars
