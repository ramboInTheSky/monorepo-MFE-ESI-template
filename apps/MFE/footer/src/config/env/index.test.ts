import env from "."

describe("ENV: ", () => {
    it("should match the SettingsSdkKeys", () => {
        expect(env).toMatchSnapshot()
    })
})
