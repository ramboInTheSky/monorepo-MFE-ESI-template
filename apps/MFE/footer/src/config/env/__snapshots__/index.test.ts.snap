// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ENV:  should match the SettingsSdkKeys 1`] = `
Object {
  "ASSETS_PATH": "/footerstatic",
  "DEVELOPMENT": false,
  "DEV_URL_OVERRIDE": "",
  "LOG_LEVEL": undefined,
  "NODE_ENV": "batman",
  "POD_NAME": undefined,
  "PORT": 3002,
  "PRODUCTION": false,
  "REACT_APP_API_BASEURL": "superman",
  "REACT_APP_APPINSIGHTS_KEY": "c620fe2b-48e7-4354-be16-d4cc42937bbd",
  "REACT_APP_BLOB_STORAGE_PATH": "/static-content",
  "REACT_APP_BLOB_STORAGE_SSR_BASEURL": "https://ecmbrowsefdsxeuw.azurefd.net",
  "REACT_APP_CDN_BASEURL": "spiderman",
  "REACT_APP_ENTRYPOINT": "/footer",
}
`;
